Bug: 1.13.3升级至1.13.4无法正常使用

### Describe the bug

升级至1.13.4后无法正常使用并报错，同时由于maxConcurrency设置为1无论隔了多久再次请求都会直接失败。

### Steps to reproduce

在1.13.4中画画 1girl

### Expected behavior

正常使用

### Screenshots

![image](https://user-images.githubusercontent.com/68104673/200813469-8e2a9def-345f-4f20-80b2-a73f065c5491.png)


### Relevant log output

```shell
2022-11-09 18:36:50 [I] app server listening at http://0.0.0.0:5141
2022-11-09 18:36:51 [W] market failed to parse @koishijs/plugin-database-sqlite
2022-11-09 18:36:51 [W] market Error: Cannot find module 'C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@minatojs\sql.js\dist\sql-wasm.js'. Please verify that the package.json has a valid "main" entry
                            at tryPackage (node:internal/modules/cjs/loader:364:19)
                            at Function.Module._findPath (node:internal/modules/cjs/loader:577:18)
                            at Function.Module._resolveFilename (node:internal/modules/cjs/loader:942:27)
                            at Function.Module._load (node:internal/modules/cjs/loader:804:27)
                            at Module.require (node:internal/modules/cjs/loader:1028:19)
                            at require (node:internal/modules/cjs/helpers:102:18)
                            at Object.<anonymous> (C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@minatojs\driver-sqlite\lib\index.js:50:26)
                            at Module._compile (node:internal/modules/cjs/loader:1126:14)
                            at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
                            at Module.load (node:internal/modules/cjs/loader:1004:32)
2022-11-09 18:36:52 [W] gocqhttp Protocol -> device lock is disable. http api may fail.
2022-11-09 18:37:34 [D] message [我的一家] 苜蓿: 画画 1girl
2022-11-09 18:37:35 [W] command novelai 1girl
                        TypeError: Cannot read properties of undefined (reading 'base64')
                            at C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\koishi-plugin-novelai\lib\index.js:236:42
                            at _Command.<anonymous> (C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\koishi-plugin-novelai\lib\index.js:256:11)
                            at processTicksAndRejections (node:internal/process/task_queues:96:5)
                            at async Array.<anonymous> (C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@koishijs\core\lib\index.cjs:1375:14)
                            at async _Command.execute (C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@koishijs\core\lib\index.cjs:1390:22)
                            at async C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@koishijs\core\lib\index.cjs:1932:22
                            at async Session.withScope (C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@koishijs\core\lib\index.cjs:1816:14)
                            at async next (C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@koishijs\core\lib\index.cjs:609:16)
                            at async next (C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@koishijs\core\lib\index.cjs:609:16)
                            at async Internal._handleMessage (C:\Users\Administrator\AppData\Roaming\Il Harper\Koishi\data\instances\default2\node_modules\@koishijs\core\lib\index.cjs:626:22)
```


### Launcher

Koishi Desktop

### Backend

NovelAI (Token)

### Versions

- OS: Windows Server 2019 Datacenter
- Adapter: onebot
- Node version: 16.18.0
- Koishi version: 4.10.0


### Additional context

_No response_

--- a/src/index.ts
+++ b/src/index.ts
@@ -232,9 +232,7 @@ export function apply(ctx: Context, config: Config) {
       const data = (() => {
         if (config.type !== 'sd-webui') {
           parameters.sampler = sampler.sd2nai(options.sampler)
-          if (image) {
-            parameters.image = image.base64 // NovelAI / NAIFU accepts bare base64 encoded image
-          }
+          parameters.image = image?.base64 // NovelAI / NAIFU accepts bare base64 encoded image
           if (config.type === 'naifu') return parameters
           return { model, input: prompt, parameters: omit(parameters, ['prompt']) }
         }

--- a/src/index.ts
+++ b/src/index.ts
@@ -232,7 +232,9 @@ export function apply(ctx: Context, config: Config) {
       const data = (() => {
         if (config.type !== 'sd-webui') {
           parameters.sampler = sampler.sd2nai(options.sampler)
-          parameters.image = image.base64 // NovelAI / NAIFU accepts bare base64 encoded image
+          if (image) {
+            parameters.image = image.base64 // NovelAI / NAIFU accepts bare base64 encoded image
+          }
           if (config.type === 'naifu') return parameters
           return { model, input: prompt, parameters: omit(parameters, ['prompt']) }
         }

